apps/api/drizzle/0002_add_edit_action.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE "public"."application_action_types" ADD VALUE 'EDIT' BEFORE 'SUBMIT_DRAFT';
```

Comment:
Debatable, but a transition from Rep Review / Dac Review to Draft due to edits should probably be tracked with an action

Reply:
As discussed, we don't need to go from Rep Review to Draft, we can allow edits to app content from the Rep Review state. If we decide in the future to capture the diff made in the rep review this will be helpful.